content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+
+* `kubelet`: komponen yang berjalan pada seluruh mesin pada klaster
+    dan memiliki tugas seperti menjalankan Pod dan kontainer.
+
+* `kubectl`: perkakas untuk berinteraksi dengan klastermu.
+
+kubeadm **tidak akan** menginstal atau mengelola `kubelet` ataupun `kubectl` untukmu, jadi kamu harus memastikan
+keduanya memiliki versi yang cocok dengan _control plane_ Kubernetes yang akan kamu instal
+dengan kubeadm. Jika tidak, ada risiko _version skew_ yang dapat terjadi dan
+dapat berujung pada perangai yang bermasalah dan tidak terduga. Namun, _satu_ _version skew_ minor antara
+kubelet dan _control plane_ masih diperbolehkan, tetapi versi kubelet tidak boleh melebihi versi API
+Server. Sebagai contoh, kubelet yang berjalan pada versi 1.7.0 akan kompatibel dengan API Server versi 1.8.0, tetapi tidak sebaliknya.
+
+Untuk informasi mengenai instalasi `kubectl`, lihat [Menginstal dan mengatur kubectl](/docs/tasks/tools/install-kubectl/).
+
+{{< warning >}}
+Instruksi ini membuat seluruh _package_ Kubernetes keluar dari _system upgrade_.
+Hal ini karena kubeadm dan Kubernetes membutuhkan
+[perhatian khusus untuk pembaharuan](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
+{{</ warning >}}
+
+Untuk informasi lebih lanjut mengenai _version skew_, lihat:
+
+* Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
+* Kubeadm-specific [version skew policy](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#version-skew-policy)
+
+{{< tabs name="k8s_install" >}}
+{{% tab name="Ubuntu, Debian atau HypriotOS" %}}
+```bash
+sudo apt-get update && sudo apt-get install -y apt-transport-https curl
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+cat <<EOF | sudo tee /etc/apt/sources.list.d/kubernetes.list
+deb https://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+sudo apt-get update
+sudo apt-get install -y kubelet kubeadm kubectl
+sudo apt-mark hold kubelet kubeadm kubectl
+```
+{{% /tab %}}
+{{% tab name="CentOS, RHEL atau Fedora" %}}
+```bash
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+exclude=kubelet kubeadm kubectl
+EOF
+
+# Mengatur SELinux menjadi permissive mode (menonaktifkannya secara efektif)
+setenforce 0
+sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
+
+yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
+
+systemctl enable --now kubelet
+```
+
+  **Catatan:**
+
+  - Mengatur SELinux menjadi _permissive mode_ dengan menjalankan `setenforce 0` dan `sed ...` menonaktifkannya secara efektif.
+    Hal ini diperlukan untuk mengizinkan kontainer untuk mengakses _filesystem_ hos, yang dibutuhkan untuk jaringan Pod sebagai contoh.
```

Comment:
```suggestion
    Hal ini diperlukan untuk mengizinkan Container untuk mengakses _filesystem_ hos, yang dibutuhkan untuk jaringan Pod sebagai contoh.
```

Reply:
Fixed in 0b74a869bea33a4eb86e43a89b5284bcd244444f